.travis.yml

```diff
@@ -13,13 +13,13 @@ before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH="$HOME/.yarn/bin:$PATH"
 
-install: yarn --frozen-lockfile --ignore-scripts
+install: yarn --frozen-lockfile
```

Comment:
This will slow down the build time, means it'll bootstrap and build the demos for all PRs

Reply:
This shouldn't be building the demos. That is under a separate npm script.

For us to check the bundle size we have to build the packages. Also, with TypeScript we will need to produce the type-definitions from the build to test and lint correctly.

This change is already merged in the `next` branch as well.

> slow down

This will slow it down, but only for the webpack output. Looks like it's only added a few minutes right now. Once we remove the deprecated packages in the next major that time should be cut in half.